portfolio/src/main/java/com/google/sps/util/SentimentAnalysisUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.util;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.io.IOException;
+import java.util.Optional;
+
+public class SentimentAnalysisUtil {
+  private final Optional<LanguageServiceClient> languageService;
+
+  public SentimentAnalysisUtil() {
+    try {
+      languageService = Optional.ofNullable(LanguageServiceClient.create());
+    } catch (IOException ioe) {
+      languageService = Optional.empty();
+    }
+  }
+
+  public float getSentimentScore(String text) {
+    float score = 0;
```

Comment:
this means that if languageService is not instantiated, the score will always be 0.
Is it intended? in this case, the error will be silently processed which is not a good idea.
I think the correct way is to raise AssertionError in the constructor when the languageService is not instantiable

Reply:
I intended for it to be this way. If I raise an assertion error then should I have a try-catch block inside the CommentDatastore class (when I try to instantiate SentimentAnalysisUtil)?